Jenkinsfile

```diff
@@ -231,13 +231,6 @@ pipeline {
                     }
                 }
 
-                stage('GCC Debug') {
```

Comment:
[todo later] Fix the issue with test_conv_igemm_dynamic_small (that takes > 4000 sec to finish -- reason unknown). Remove `codecov`. Restore this stage after that.

Reply:
![image](https://user-images.githubusercontent.com/12697412/99197265-7d947a80-27a2-11eb-9e23-cd7424c5fc16.png)